tests/js/list-tests.js

```diff
@@ -1475,6 +1475,28 @@ module.exports = {
             TestCase.assertEqual(addresses[i]["street"], streets[i]);
         }
 
+        // creating standalone embedded object is not allowed
+        let addrs1 = owners[0].addresses;
+        realm.write(() => {
+            TestCase.assertThrows(() => {
+                realm.create(schemas.AddressSchema.name, { street: "Njalsgade", city: "Islands Brygge" });
+            });
+
+            TestCase.assertThrows(() => {
+                addrs1.push({ street: "Njalsgade", city: "Islands Brygge" });
```

Comment:
We're not allowed to push to an existing user's list of addresses either?

Reply:
The above implies that you create an embedded object. Standalone or free-floating embedded objects are not allowed. Moreover, you cannot move an embedded object (removing it from it parent implies that it becomes free-floating).

Reply:
See "Semantics" in https://github.com/realm/realm-java/pull/6730

Reply:
But wouldn't this be equal to:
```java
managedObject.list.add(EmbeddedPerson()) // Will automatically copy the object as embedded
```
 (line from description @ https://github.com/realm/realm-java/pull/6730)

Reply:
Fine as a temp crash fix.

Reply:
We will reintroduce (fix might be a better word) the ability to push to a list later.